package.json

```diff
@@ -23,5 +21,11 @@
     "dev": "encore dev",
     "watch": "encore dev --watch",
     "build": "encore production --progress"
+  },
+  "dependencies": {
+    "add": "^2.0.6",
```

Comment:
what is this dependency ? I suspect you ran `yarn add yarn add` by mistake.

Reply:
you're totally right, i remove `add` and `yarn` dependencies